test/e2e/testdata/workflow-template-with-containerset.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: argoproj.io/v1alpha1
+kind: WorkflowTemplate
+metadata:
+  name: containerset-with-retrystrategy
+  annotations:
+    workflows.argoproj.io/description: |
+      This workflow template is used to create a workflow with containerset.
```

```suggestion
      This workflow template creates a container set with retryStrategy.
```

- simpler and more direct, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)
- same as above